tests/LoggedIn/Loans/request-loan.spec.js

```diff
@@ -0,0 +1,23 @@
+import { test } from '../../_fixtures/fixtures';
+import * as allure from 'allure-js-commons';
+import { NavBar } from '../../../src/ui/components/NavBar.js';
+import { RequestLoanPage } from '../../../src/ui/pages/RequestLoanPage.js';
+
+const cases = [
+  { amount: '100',  down: '10',  severity: 'normal'   },
+  { amount: '5000', down: '100', severity: 'critical' },
+  { amount: '0',    down: '0',   severity: 'minor'    },
+];
+
+for (const c of cases) {
```

Comment:
```suggestion
cases.forEach(({ amount, down, severity }) => {
```

https://playwright.dev/docs/test-parameterize